### PR TITLE
fix(DB/Creature): Gyromasts Revenge

### PR DESCRIPTION
**Pull request by wizzymore**
## Changes Proposed:
-  Move The Threshwackonator 4100 scripts from C++ to SAI
-  Correct the gossip menus that the creature uses
- Correct the script of the npc.

## Issues Addressed:
- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/4099

## Tests Performed:
- Ubuntu 20.04

## How to Test the Changes:

.go c id 6669
Right click the NPC and observe the menu
.q a 2078
Right click the NPC and observe the menu now
Click the gossip option and complete the quest to check the script.

## Target Branch(es):
- [x] Master

 
## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here in the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

### CODE REVIEW

File: data/sql/updates/pending_db_world/rev_1609354039936904092.sql [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+INSERT INTO `conditions` (`SourceTypeOrReferenceId`, `SourceGroup`, `SourceEntry`, `SourceId`, `ElseGroup`, `ConditionTypeOrReference`, `ConditionTarget`, `ConditionValue1`, `ConditionValue2`, `ConditionValue3`, `NegativeCondition`, `ErrorType`, `ErrorTextId`, `ScriptName`, `Comment`) VALUES
+(22, 1, 6669, 0, 0, 9, 0, 2078, 0, 0, 1, 0, 0, '', 'The Threshwackonator 4000 - Default Gossip');
+
+DELETE FROM `conditions` WHERE (`SourceTypeOrReferenceId` = 22) AND (`SourceGroup` = 2) AND (`SourceEntry` = 6669) AND (`SourceId` = 0) AND (`ElseGroup` = 0) AND (`ConditionTypeOrReference` = 9) AND (`ConditionTarget` = 0) AND (`ConditionValue1` = 2078) AND (`ConditionValue2` = 0) AND (`ConditionValue3` = 0);
+INSERT INTO `conditions` (`SourceTypeOrReferenceId`, `SourceGroup`, `SourceEntry`, `SourceId`, `ElseGroup`, `ConditionTypeOrReference`, `ConditionTarget`, `ConditionValue1`, `ConditionValue2`, `ConditionValue3`, `NegativeCondition`, `ErrorType`, `ErrorTextId`, `ScriptName`, `Comment`) VALUES
+(22, 2, 6669, 0, 0, 9, 0, 2078, 0, 0, 0, 0, 0, '', 'The Threshwackonator 4000 - Gossip when Quest ID: 2078 taken');
```
Comment (Kitzunu) [inline data/sql/updates/pending_db_world/rev_1609354039936904092.sql:9]: You can do this in one delete SourceGroup IN (1,2)

Comment (wizzymore) [reply data/sql/updates/pending_db_world/rev_1609354039936904092.sql]: Done, thanks :D

**Review by FrancescoBorzi — APPROVED** [review]

**Comment by wizzymore** [comment]
@pklloveyou Hi, thanks for the report, this is now fixed.

**Comment by pklloveyou** [comment]
image
Tested

**Comment by pklloveyou** [comment]
The task cannot be completed...

**Comment by wizzymore** [comment]
@pklloveyou yep, seems like when i tested i always killed him faster than the evade time, now it should work as intended, the place where he becomes hostile becomes his new home, if OOC he will evade to the place where he got hostile for 30 seconds, then go back to the spawn with the correct flags and faction. ( So other players can re-complete the quest if the npc killed a player or the player DC'ed or ran away )

**Comment by pklloveyou** [comment]
Tested perfect

**Comment by pklloveyou** [comment]
I'm going to sleep, good night

**Comment by LukasVolgger** [comment]
Thx @pklloveyou and @wizzymore ! Have a good night!

**Comment by Kitzunu** [comment]
Thanks!

**Pull request merged**